src/app/app.component.scss

```diff
@@ -411,3 +411,53 @@
 ::ng-deep.cdk-global-scrollblock {
   overflow-y: hidden !important;
 }
+
+.shortcut-list {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  font-size: $default-font-size;
+  height: $filter-height;
+  z-index: 2;
+
+  .shortcut {
+    padding: 0 $sm-spacing;
+    height: fit-content;
+
+    // Tighten powerbars display
+    ::ng-deep.checkbox {
+      padding-bottom: 0;
+    }
+
+    // Manage shortcuts header height to fit them in top menu
+    ::ng-deep.title {
+      height: $filter-height;
+    }
+
+    ::ng-deep.content {
+      // Allow enough space for the widget and its padding
+      // width: 258px !important;
```

Comment:
Needed or not ?

Reply:
No it is not, thanks to the fix that was made before on the box-sizing